tests/sentry/incidents/endpoints/test_project_alert_rule_details.py

```diff
@@ -106,7 +106,9 @@ def test_permissions(self):
     def test_no_feature(self):
         self.login_as(self.owner_user)
         resp = self.get_response(self.organization.slug, self.project.slug, self.alert_rule.id)
-        assert resp.status_code == 404
+        # Without incidents feature flag, allow delete
+        status = 204 if self.method == "delete" else 404
```

Comment:
Perhaps I'm missing it, but does the `delete` branch of this logic ever get run by the test?

Reply:
it does, there's a delete test suite that inherits from this one